params/protocol_params.go

```diff
@@ -167,12 +167,13 @@ const (
 	BlobTxBytesPerFieldElement         = 32      // Size in bytes of a field element
 	BlobTxFieldElementsPerBlob         = 4096    // Number of field elements stored in a single data blob
 	BlobTxHashVersion                  = 0x01    // Version byte of the commitment hash
-	MaxBlobGasPerBlock                 = 1 << 19 // Maximum consumable blob gas for data blobs per block
-	BlobTxTargetBlobGasPerBlock        = 1 << 18 // Target consumable blob gas for data blobs per block (for 1559-like pricing)
 	BlobTxBlobGasPerBlob               = 1 << 17 // Gas consumption of a single data blob (== blob byte size)
 	BlobTxMinBlobGasprice              = 1       // Minimum gas price for data blobs
-	BlobTxBlobGaspriceUpdateFraction   = 2225652 // Controls the maximum rate of change for blob gas price
+	BlobTxBlobGaspriceUpdateFraction   = 3338477 // Controls the maximum rate of change for blob gas price
 	BlobTxPointEvaluationPrecompileGas = 50000   // Gas price for the point evaluation precompile.
+
+	MaxBlobGasPerBlock    = 6 * BlobTxBlobGasPerBlob // Maximum consumable blob gas for data blobs per block
+	TargetBlobGasPerBlock = 3 * BlobTxBlobGasPerBlob // Target consumable blob gas for data blobs per block (for 1559-like pricing)
```

Comment:
The params is already a huge collection of completely random names stuff that you have no idea where they come from. I.e. RefundQuotient, which EIP does it belong to? Nobody knows unless you really start digging deep. The BlobTx prefix was meant to follow the Bla12381 prefix idea to group the constants into more meaningful groups, even if a bit too verbose. I'd highly recommend keeping the prefix.

Reply:
<img width="530" alt="Screenshot 2023-08-30 at 09 57 28" src="https://github.com/ethereum/go-ethereum/assets/129561/e1fa4c6b-228e-4684-8f90-57f661d60fa8">

Reply:
I can open a separate PR to adjust the naming. The change has been reverted in this PR.